src/components/User.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+export function User({ user }) {
```

Comment:
``` export function User({ name, email, address: { city, street, zipcode} }) {```

Reply:
It's better to avoid more than one level of destructuring.